### PR TITLE
HDPI-5731: update notice details screen

### PR DESCRIPTION
**Pull request by gmmagruder**
### Jira link

See [HDPI-5731](https://tools.hmcts.net/jira/browse/HDPI-5731)

### Change description

- Updated content for email and other options in notice details page
- Renamed noticeEmailExplanation to noticeEmailAddress
- Changed input for noticeEmailAddress from multi line textbox to single line textbox and reduced character limit to 60

### Testing done

Manual testing locally

### Security Vulnerability Assessment ###

**CVE Suppression:** Are there any CVEs present in the codebase (either newly introduced or pre-existing) that are being intentionally suppressed or ignored by this commit?
  * [ ] Yes
  * [x] No

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/pcs/ccd/domain/NoticeServedDetails.java [hunk 1]
```diff
@@ -80,15 +80,13 @@ public class NoticeServedDetails {
     private String noticePersonName;
 
     @CCD(
-            label = "Explain how it was served by email",
-            hint = "You can enter up to 250 characters",
-            typeOverride = TextArea
+            label = "What email address was the document sent to?",
+            hint = "For example, name@example.com"
```
Comment (scottstewart-sl) [inline src/main/java/uk/gov/hmcts/reform/pcs/ccd/domain/NoticeServedDetails.java]: Maybe add `typeOverride = FieldType.Email` to get some validation of the input value, (if that is what you want).

Also since this is now a single line text field, you should hopefully just be able to add the `max` attribute on here and ExUI/CCD will do the length validation for you and you wouldn't need the custom validation in `NoticeDetailsService.java` . (Give that a quick test though to make sure it works with the Email type, if you do use that).

File: src/test/java/uk/gov/hmcts/reform/pcs/ccd/service/NoticeDetailsServiceTest.java [hunk 2]
```diff
@@ -378,19 +378,19 @@ void shouldValidateEmailWithValidExplanation() {
...
         @Test
-        void shouldValidateEmailWithTooLongExplanation() {
+        void shouldValidateEmailWithTooLongEmailAddress() {
             // Given
             LocalDateTime pastDateTime = LocalDateTime.now().minusDays(1);
 
-            // Create a string that exceeds 250 characters
-            String longText = "0123456789".repeat(26); // 10 chars x 26 = 260 chars
+            // Create a string that exceeds 60 characters
+            String longText = "0123456789".repeat(7); // 10 chars x 7 = 70 chars
```
Comment (scottstewart-sl) [inline src/test/java/uk/gov/hmcts/reform/pcs/ccd/service/NoticeDetailsServiceTest.java]: I think 61 would be a better test since it is the boundary condition.

File: src/main/java/uk/gov/hmcts/reform/pcs/ccd/domain/NoticeServedDetails.java [hunk 3]
```diff
@@ -80,15 +80,13 @@ public class NoticeServedDetails {
...
     )
-    private String noticeEmailExplanation;
+    private String noticeEmailAddress;
```
Comment (guygrewal77) [inline src/main/java/uk/gov/hmcts/reform/pcs/ccd/domain/NoticeServedDetails.java:89]: Could this name change cause an issue with elastic search in the master build ? Or can an elastic search reindex be carried out somehow ? Might be worth checking with Praveen on this.

Comment (scottstewart-sl) [reply src/main/java/uk/gov/hmcts/reform/pcs/ccd/domain/NoticeServedDetails.java:89]: Hopefully a name change would be OK, as long as it is not a name that has already been used with a different type. I think we can try this one with crossed fingers, rather than going to the effort of deprecating the old field name, deploying, and then removing the old field and deploying again.

If there is an issue with the build pipeline that mixes the staging and AAT envs then we should try to fix that instead of working around it.

**Review by guygrewal77 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/pcs/ccd/domain/NoticeServedDetails.java (hunk 1 above)
Comment (gmmagruder) [reply src/main/java/uk/gov/hmcts/reform/pcs/ccd/domain/NoticeServedDetails.java]: Updated to use typeOverride = FieldType.Email. Tested locally with max = 60 and ExUI validation was triggered.

File: src/test/java/uk/gov/hmcts/reform/pcs/ccd/service/NoticeDetailsServiceTest.java (hunk 2 above)
Comment (gmmagruder) [reply src/test/java/uk/gov/hmcts/reform/pcs/ccd/service/NoticeDetailsServiceTest.java]: No longer needed as validation is done on the ExUI side.

**Review by scottstewart-sl — APPROVED** [review]
